recipes/xcfun/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "xcfun" %}
+{% set version = "2.0.0a4" %}
+{% set sha256 = "f4a013710c24d0d0c519043e7210fdecb8d7376b65ca9b4d5431da9b8785621c" %}
+{% set pyver = PY_VER.replace(".", "") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dftlibs/xcfun/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  skip: true  # [py2k]
+  binary_relocation: true
+  run_exports:
+    - {{ pin_subpackage('xcfun', max_pin='x') }}
+
+requirements:
+  build:
+    - cmake >=3.5
+    - {{ compiler('cxx') }}
+    - pybind11 =*=*py{{ pyver }}*
+    - ninja
+  host:
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  requires:
+    - numpy
+    - pytest
+  imports:
+    - xcfun
+  commands:
+    # Verify library
+    - test -L $PREFIX/lib/libxcfun$SHLIB_EXT            # [unix]
+    - test -f $SP_DIR/xcfun/_xcfun*$SHLIB_EXT           # [unix]
+    - if not exist %PREFIX%\\Library\\lib\\xcfun.lib exit 1  # [win]
+    - if not exist %SP_DIR%\\xcfun\\_xcfun*.dll exit 1  # [win]
+    # Inspect linkage
+    - ldd -v $PREFIX/lib/libxcfun$SHLIB_EXT                           # [linux]
+    - otool -L $PREFIX/lib/libxcfun$SHLIB_EXT                         # [osx]
+    - conda inspect linkages --show-files --groupby=dependency $PKG_NAME  # [unix]
+    - conda inspect objects -p $PREFIX $PKG_NAME                      # [osx]
+    - ls -l $SP_DIR/xcfun  # [unix]
+    - dir %SP_DIR%\xcfun   # [win]
+    - ldd -v $SP_DIR/xcfun/_xcfun*$SHLIB_EXT    # [linux]
+    - otool -L $SP_DIR/xcfun/_xcfun*$SHLIB_EXT  # [osx]
+    # Actually test
+    - python -m pytest --pyargs xcfun
+
+outputs:
+  - name: xcfun
+    files:
+      - include/XCFun              # [unix]
+      - lib/libxcfun*              # [unix]
+      - share/cmake/XCFun          # [unix]
+      - Library/include/XCFun      # [win]
+      - Library/lib/xcfun*         # [win]
+      - Library/share/cmake/XCFun  # [win]
+      - lib/python{{ PY_VER }}/site-packages/xcfun  # [unix]
+      - Lib/site-packages/xcfun                     # [win]
+
+about:
+  home: https://github.com/dftlibs/xcfun
+  dev_url: https://github.com/dftlibs/xcfun
+  doc_url: https://xcfun.readthedocs.io
+  doc_source_url: https://github.com/dftlibs/xcfun/blob/master/doc/index.rst
+  license: MPL-2.0
+  license_url: https://opensource.org/licenses/MPL-2.0
+  license_file: LICENSE.md
+  license_family: MOZILLA
+  summary: "Exchange-Correlation functionals with arbitrary order derivatives."
+
+extra:
+  recipe-maintainers:
+    - robertodr
+    - bast
```

Comment:
@bast are you willing to be a maintainer?

Reply:
Yes